applications/compareSummary.cpp

```diff
@@ -0,0 +1,460 @@
+/*
+  Copyright 2016 Statoil ASA.
+  This file is part of the Open Porous Media project (OPM).
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
```

Comment:
this is a bit nitpicking that I don't really care that much about, but why didn't you use the default header?  See  [Cells.hpp](https://github.com/OPM/opm-output/blob/master/opm/output/Cells.hpp)